components/Buttons/DeleteCommentButton/index.js

```diff
@@ -2,7 +2,7 @@ import { styled } from "styled-components";
 
 export default function DeleteCommentButton({ onClick }) {
   return (
-    <StyledButton onClick={onClick}>
+    <StyledButton onClick={onClick} aria-label="Delete Comment Button">
```

Comment:
when the screenreader reads the page, it will already announce that its a button because of the HTML Element `<button>`, therefore it is recommended to add an aria label without the word "button", like this: 
```suggestion
    <StyledButton onClick={onClick} aria-label="Delete Comment">
```
or maybe 
```suggestion
    <StyledButton onClick={onClick} aria-label="click to delete comment">
```